tests/Etc/Comment.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Stancl\Tenancy\Tests\Etc;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Comment extends Model
```

Comment:
Do models (`Comment`) that belong to models (`Post`) that belong to a tenant not need any traits?

Reply:
Ah I see that this is what scoped comments do. Probably add comments explaining the difference then, since from `Post` it's not clear that it's not scoped.

Reply:
I made [some small changes](https://github.com/archtechx/tenancy/pull/1105/commits/1f9a344f33e158edf6aa3283d2e81c7670d30339). I think the difference is clearer now

Reply:
> This model is not intended to be used with the single-database tenancy approach

Rather than mentioning that something is not intended to be used with a certain thing, comments should explain what the thing *is* used for.

Reply:
Yeah, that makes sense. I'll update that

Reply:
https://github.com/archtechx/tenancy/pull/1105/commits/28efc0ca97e41e275a9e4d49ac304e8c06d86127 updated the docblocks